appserver/extras/docker-images/pom.xml

```diff
@@ -64,7 +63,7 @@
         <docker.payara.rootDirectoryName>payara5</docker.payara.rootDirectoryName>
 
         <docker.payara.repository>payara/${project.artifactId}</docker.payara.repository>
-        <docker.payara.tag>${payara.version}</docker.payara.tag>
+        <docker.payara.tag>${project.version}</docker.payara.tag>
```

Comment:
This would screw up maintenance of previous Docker images.
We don't backport every single change to our Docker images to previous tags & branches - we just build from the master branch and specify the version of Payara to bundle.